src/vaadin-grid-column-group.html

```diff
@@ -104,6 +108,7 @@
           this._observer && this._observer.disconnect();
         }
 
+        /** @private */
         _columnPropChanged(path, value) {
```

Comment:
Should this be `@protected` since it's called from many places in `vaadin-grid-column.js` too?

Reply:
Good catch, done.